### DISCUSSION TITLE
Support for .Net Core 3.1

### DISCUSSION THREAD

**Discussion by FastTracOnline** [Megabit/Blazorise#2471] [Q&A]
Looking at the various nuGet versions for Blazorise.Bootstrap, I noticed that support for .Net Core 3.1 has been discontinued as of v0.9.3 with the removal of .Net Standard 2.1 support.

Unfortunately, finding good web hosts out there that support .Net 5 has been a challenge.  And no, I won't use Azure and be completely dependent on Microsoft's goodwill.  The state of .Net hosting seems to be 3.1 and if a library does not support that I cannot use it in good conscience.  Eventually the hosts will catch up, but that time is not yet.

**Comment by stsrki** [comment]
Unfortunately there is no way back. Dropping the support for 3.1 was the only option because we needed to support ComponentActivator. Before 0.9.3 every custom Blazorise component had to be rendered two times, which lead to complicate component tree, memory leaks, and slower runtime. With CA all of that was finally solved. And supporting two frameworks would require too much resources. So I had to made a choice. Blazorise is now faster, easier to maintain and uses much less memory. Not to mention less spagheti code.

I know it is not the answer you wanted to hear...

**Reply by FastTracOnline** [reply]
I understand why you did it but if I cannot find a hosting service that can handle it.... no site, no need for a component.  Real world for the small guys.

**Comment by pekatete** [comment]
You can get a single core, 20GB  VPS for 2 EURO a month – sufficient for hosting multiple websites.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Reply by stsrki** [reply]
I just google it and there are options https://www.webwiz.net/hosting/asp.net-5-hosting.htm
